### PR TITLE
[tickets] fix: improve purchase tickets UX when VSP listing is disabled on privacy settings

### LINKED ISSUE #2896: Privacy settings on purchase ticket form
When privacy is enabled, the full privacy settings are shown on the purchase ticket form. I suggest removing this, and instead displaying a plain text warning when the mixed account is being used to purchase tickets, something like "You are purchasing mixed tickets."

### LINKED ISSUE #2921: "Options not defined" when VSP listing is disabled
When "VSP Listing" is disabled on the Settings tab, the VSP form looks like the screenshot below. Ideally we should inform the user they need to change their settings, and maybe provide a button which changes it, similar to the governance tab when the politeia integration is disabled.

Screenshot from 2020-11-12 15-19-07

### PR DESCRIPTION
**Pull request by victorgcramos**
This Diff adds some tweaks on purchase tickets form, telling user that if `VSP Listing` is disabled by privacy settings, users won't be able to use VSPs to purchase tickets, since it blocks external requests.

Closes #2921

### CODE REVIEW

File: app/actions/ControlActions.js [hunk 1]
```diff
@@ -360,13 +358,12 @@ export const newPurchaseTicketsAttempt = (
...
     const dontSignTx = sel.isWatchingOnly(getState());
     dispatch({ numTicketsToBuy: numTickets, type: PURCHASETICKETS_ATTEMPT });
-
     const csppReq = {
       mixedAccount: sel.getMixedAccount(getState()),
       changeAccount: sel.getChangeAccount(getState()),
       csppServer: sel.getCsppServer(getState()),
       csppPort: sel.getCsppPort(getState()),
-      mixedAcctBranch: sel.getMixedAccountBranch(getState())
+      mixedAcctBranch: 0
```
Comment (amassarwi) [inline app/actions/ControlActions.js:366]: this was fixed on master - please rebase

**Review by vctt94 — COMMENTED** [review]
Can you open a new PR for each issue. This way it is easier to review and keep things separated and smaller.

**Comment by victorgcramos** [comment]
@vctt94 done, mate! thanks for the suggestion!

**Review by jholdstock — CHANGES_REQUESTED** [review]
The warning and the button appear at the very bottom of the page. I actually had to scroll down before I could see them. Very easy to miss. I think they should be more prominent, probably above the "Purchase Tickets" header.

Screenshot from 2020-11-20 10-00-47

File: app/components/views/TicketsPage/PurchaseTab/PurchaseTickets/Page.jsx [hunk 2]
```diff
@@ -15,25 +16,44 @@ export const LegacyVSPWarning = () => (
...
+    </Tooltip>
+  </div>
+);
+
+const EnableVSP = ({ onEnableVSP }) => (
+  <div className={styles.enableVSPWrapper}>
+    <p>
+      <T
+        id="purchase.vsp.enableListing.description"
+        m="VSP Listing is currently disabled in your privacy settings. Please enable it if you want to be able to purchase tickets from listed VSPs."
```
[the remark this answers is not in the document]
Comment (victorgcramos) [reply app/components/views/TicketsPage/PurchaseTab/PurchaseTickets/Page.jsx:50]: Oh, sure! Thanks!

**Comment by jholdstock** [comment]
Needs another rebase

**Comment by jholdstock** [comment]
Trying this now, and I see the button in the new location, it looks much better. However it seems like nothing is happening when I press the button.

File: app/components/views/TicketsPage/PurchaseTab/PurchaseTab.module.css [hunk 3]
```diff
@@ -87,9 +87,19 @@
...
+.enableVSPButton {
+  margin-bottom: 40px;
+}
+
+.enableVSPWrapper {
+  margin: 10px;
+}
+
+.enableVSPMessage {
+  background-image: var(--info-warning-icon);
```
Comment (jholdstock) [inline app/components/views/TicketsPage/PurchaseTab/PurchaseTab.module.css:99]: I dont see this icon anywhere?

Screenshot from 2020-12-03 15-13-03

**Review by jholdstock — COMMENTED** [review]
Functionality is fixed now - just one outstanding comment about a missing icon

File: app/components/views/TicketsPage/PurchaseTab/PurchaseTab.module.css (hunk 3 above)
Comment (victorgcramos) [reply app/components/views/TicketsPage/PurchaseTab/PurchaseTab.module.css]: @jholdstock I think the icon makes no sense here. Instead, I fixed some minor styling issues regarding the enable message. Thanks for the suggestion!

**Comment by jholdstock** [comment]
Unsure if it is related to your PR, but I am seeing a random `0` under the Purchase Tickets heading 

Screenshot from 2020-12-04 09-52-37

**Review by vctt94 — APPROVED** [review]
LGTM.

I am not being able to reproduce this 0 bug @jholdstock mentioned.

**Review by jholdstock — APPROVED** [review]
The zero was appearing because I was using a misconfigured wallet (account index 0 set as unmixed account #3033)

Using a properly configured wallet, this PR looks good.

**Pull request merged**